Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -57,34 +59,35 @@ namespace gdjs {
       for (let i = 0; i < this._borderSprites.length; ++i) {
         this._spritesContainer.addChild(this._borderSprites[i]);
       }
-      this._alpha = this._spritesContainer.alpha;
+      this._wrapperContainer.addChild(this._spritesContainer);
+      this._alpha = this._wrapperContainer.alpha;
       runtimeScene
         .getLayer('')
         .getRenderer()
-        .addRendererObject(this._spritesContainer, runtimeObject.getZOrder());
+        .addRendererObject(this._wrapperContainer, runtimeObject.getZOrder());
     }
 
     getRendererObject() {
-      return this._spritesContainer;
+      return this._wrapperContainer;
     }
 
     ensureUpToDate() {
       if (this._spritesContainer.visible && this._wasRendered) {
         // Update the alpha of the container to make sure that it's applied.
         // If not done, the alpha will be back to full opaque when changing the color
         // of the object.
-        this._spritesContainer.alpha = this._alpha;
+        this._wrapperContainer.alpha = this._alpha;
```

Comment:
I *think* we can remove this assignment to alpha altogether.
It was there before to workaround a similar bug related to the alpha being overwritten when we were tweaking the color of the object... but because we now change the alpha of the wrapper container, we should be 100% safe?

You can then remove it, remove the _alpha property, and just add a comment about the `cacheAsBitmap` to emphasize the workaround again:

```
// Cache the rendered sprites as a bitmap to speed up rendering when lots of panel sprites are on the scene.
// Sadly, because of this, we need a wrapper container to workaround a PixiJS issue with alpha (see updateOpacity).
```

Reply:
Is it still relevant to have the ensureUpToDate function at all?
As this function is called for every loop of the game engine, it feels overkill to set `cacheAsBitmap=true` on every frame

Reply:
Setting cacheAsBitmap to true is actually calling a "magic" setter (not a big fan of these hidden API calls, but that's how PixiJS works): https://github.com/pixijs/pixijs/blob/f1d98ea807b8bbe7f3aaf41e05f37cf219e79994/packages/mixin-cache-as-bitmap/src/index.ts#L135-L145

It will quickly bail out if already set to true. Otherwise, it will actually generate the cached bitmap at this moment - I think it's important to do it here? We could check if it was already cached, but that's what the setter is doing. So I would say to keep it? 
`cacheAsBitmap` is set to false in other part of the code that can modify the rendered image (namely: changed tint color or changed width/height/texture).